### DISCUSSION TITLE
Unable to push commit to git

### DISCUSSION THREAD

**Discussion by toanparadox** [botfront/botfront#967] [Q&A]
Hi, I'm working on integrate Botfront with GitHub, it seems have some issue, when I setting the branch name in git settings:
- For branch `master`: `git@github.com:toanparadox/rasa_nlp.git#master`:  Able to connect, commit and push 
- I create a branch `test` from `master`: `git@github.com:toanparadox/rasa_nlp.git#test`: Can't connect, it shows `Could not connect to branch test on your git repo. Check your credentials`
- When I add `origin` before branch name:  `git@github.com:toanparadox/rasa_nlp.git#origin/test`, I can retrieve git history but can't commit and push. 

Can anyone help with this issue?

**Comment by pheel** [comment]
Hi @toanparadox,
Should be fixed in 1.0.3 with https://github.com/botfront/botfront/pull/953/.
